### ISSUE TITLE
Revoked certificate is flagged as valid

### ISSUE THREAD

**Issue by sorcio46** [dyaa/ssl-checker#251]
I was testing this library with the following use case:

- [ERR_CERT_REVOKED](https://revoked.badssl.com/)

I was expecting the result of this test to be invalid due to the error _ERR_CERT_REVOKED_ active on the certificate but this is what I got:
```json
{
    "daysRemaining": 352,
    "valid": true,
    "validFrom": "2021-10-27T00:00:00.000Z",
    "validTo": "2022-10-27T23:59:59.000Z",
    "validFor": [
        "revoked.badssl.com"
    ]
}
```

**Comment by mitikov** [comment]
Here are some tests proving it does not work [[runkit](https://runkit.com/mitikov/ssl-cert-tests/2.1.0)]
